core/commands/refs.go

```diff
@@ -50,9 +50,9 @@ Note: List all references recursively by using the flag '-r'.
 	},
 	Options: []cmds.Option{
 		cmds.StringOption("format", "Emit edges with given format. Available tokens: <src> <dst> <linkname>."),
-		cmds.BoolOption("edges", "e", "Emit edge format: `<from> -> <to>`."),
-		cmds.BoolOption("unique", "u", "Omit duplicate refs from output."),
-		cmds.BoolOption("recursive", "r", "Recursively list links of child nodes."),
+		cmds.BoolOption("edges", "e", "Emit edge format.").Default("`<from> -> <to>`"),
```

Comment:
This option is a boolean, you can't use a string for a default value (see test failures)